### PR TITLE
[LMS Shift] Grand Finals update

### PR DESCRIPTION
**Pull request by fajar13k**
### Description
- Update addition of the tournamen to Grand Finals (Schedules, Results, and Mappool)
- Some Better wording
- Spacing fix

### Status
Pending Review // Finished

### CODE REVIEW

File: wiki/Tournaments/LMS/Shift/1/en.md [hunk 1]
```diff
@@ -33,22 +33,22 @@ Taiko LMS: Shift Cup
 
  *The remaining prizes are to be decided.*
 
-## Organization
+## Organisation
```
Comment (TPGPL) [inline wiki/Tournaments/LMS/Shift/1/en.md]: could be left as Organization

File: wiki/Tournaments/LMS/Shift/1/en.md [hunk 2]
```diff
@@ -33,22 +33,22 @@ Taiko LMS: Shift Cup
...
-| Commentators | ![][flag_US] [Backfire](https://osu.ppy.sh/users/263110), ![][flag_GB] [Grimbow](https://osu.ppy.sh/users/1387762), ![][flag_US] [Binsu](https://osu.ppy.sh/users/6236911), ![][flag_HK] [akali393394](https://osu.ppy.sh/users/9686628), ![][flag_TR] [frukoyurdakul](https://osu.ppy.sh/users/7612550), ![][flag_AU] [Jaye](https://osu.ppy.sh/users/4841352) |
-| Streamers | ![][flag_DE] [Heamburger](https://osu.ppy.sh/users/4705120), ![][flag_US] [Binsu](https://osu.ppy.sh/users/6236911) |
-| Designer | ![][flag_FI] [Micro](https://osu.ppy.sh/users/4801080) |
-| Referees | ![][flag_AU] [cactusthegreat](https://osu.ppy.sh/users/5592409), ![][flag_GB] [Grimbow](https://osu.ppy.sh/users/1387762), ![][flag_SE] [Eiuh](https://osu.ppy.sh/users/4653213), ![][flag_CA] [janitoreihil](https://osu.ppy.sh/users/3307897), ![][flag_DE] [lukili846](https://osu.ppy.sh/users/6278008) |
+| Management and Organisation | ![][flag_US] [Backfire](/users/263110), ![][flag_CA] [FlamingRok](/users/2015353), ![][flag_GB] [Lno](/users/1900787) |
```
Comment (TPGPL) [inline wiki/Tournaments/LMS/Shift/1/en.md]: `Management` itself might be enough?

File: wiki/Tournaments/LMS/Shift/1/en.md [hunk 3]
```diff
@@ -33,22 +33,22 @@ Taiko LMS: Shift Cup
...
+| Map Selectors | ![][flag_US] [Backfire](/users/263110), ![][flag_PT] [babysnakes](/users/4669728), ![][flag_GB] [Lno](/users/1900787) |
+| Commentators | ![][flag_US] [Backfire](/users/263110), ![][flag_GB] [Grimbow](/users/1387762), ![][flag_US] [Binsu](/users/6236911), ![][flag_HK] [akali393394](/users/9686628), ![][flag_TR] [frukoyurdakul](/users/7612550), ![][flag_AU] [Jaye](/users/4841352) |
+| Streamers | ![][flag_DE] [Heamburger](/users/4705120), ![][flag_US] [Binsu](/users/6236911) |
+| Designer | ![][flag_FI] [Micro](/users/4801080) |
+| Referees | ![][flag_AU] [cactusthegreat](/users/5592409), ![][flag_GB] [Grimbow](/users/1387762), ![][flag_SE] [Eiuh](/users/4653213), ![][flag_CA] [janitoreihil](/users/3307897), ![][flag_DE] [lukili846](/users/6278008) |
 
 ## Links
 
-- [Official Forums Thread](https://osu.ppy.sh/community/forums/topics/744163)
+- [Official Forums Thread](/community/forums/topics/744163)
```
Comment (TPGPL) [inline wiki/Tournaments/LMS/Shift/1/en.md]: Official Forums -> Discussion

File: wiki/Tournaments/LMS/Shift/1/en.md (hunk 1 above)
Comment (Joehuu) [reply wiki/Tournaments/LMS/Shift/1/en.md]: Do we use `Organization` and forget about the British English @TPGPL?

**Comment by peppy** [comment]
There's no need for the headers or "review" status in the PR description.

**Pull request merged**